src/node_kernels/TKEKsgsNodeKernel.C

```diff
@@ -42,9 +42,6 @@ TKEKsgsNodeKernel::setup(Realm& realm)
   dudx_            = fieldMgr.get_field<double>(dudxID_);
   dualNodalVolume_ = fieldMgr.get_field<double>(dualNodalVolumeID_);
 
-  const std::string dofName = "turbulent_ke";
```

Comment:
Lol so this was never used? 🤦‍♂️

Reply:
yup... it comes down to copy-paste software development I think ;)